packages/microsoft-reactnative-sampleapps/windows/SampleAppCS/SampleAppCS.csproj

```diff
@@ -19,7 +20,7 @@
     <WindowsXamlEnableOverview>true</WindowsXamlEnableOverview>
     <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
     <AppxGeneratePrisForPortableLibrariesEnabled>false</AppxGeneratePrisForPortableLibrariesEnabled>
-    <LangVersion>7.0</LangVersion>
+    <LangVersion>8.0</LangVersion>
```

Comment:
>8.0 [](start = 17, length = 3)

Does it work?I would expect that min version of .Net Native requires us to use 7.3.

Reply:
This seems to works fine. The only things that wont work are default interfaces and index and range expressions.

I can place it back to 7.3 if you prefer.


---
In reply to: [420382298](https://github.com/microsoft/react-native-windows/pull/4793#discussion_r420382298) [](ancestors = 420382298)

Reply:
No, as long it compiles and runs in .Net Native (samples Release mode) we should be good.  It is exciting to be able to use latest version of the language!

Reply:
@dannyvv please revert back to 7 - WinUI 3 doesn't yet support C# 8:
https://docs.microsoft.com/en-us/uwp/toolkits/winui3/#visual-studio-project-templates

Reply:
@asklar, will it work with C# 7.3? I am also confused by them saying that it supports UWP, but does not support .Net Core. The UWP only uses .Net Core. What to believe?